content/tutorials/Super-lightweight-realtime-subscriptions-with-SSE-and-Ably.textile

```diff
@@ -0,0 +1,223 @@
+              background-color: #333;
+              height: 300px;
+              overflow: scroll;
+              color: orange;
+          }
+          h1{
+            text-align: center;
+          }
+          h3{
+            text-align: center;
+          }
+    </style>
+  </head>
+  <body>
+    <h1>SSE example</h1>
+
+    <h3>Events:</h3>
+    <ul id="events"></ul>
+  </body>
+</html
+```
+
+What's happening in the code above? We create a basic HTML structure which consists of an @ul@ element. The children of this element will be dynamically populated.
+
+*Setting up the Publisher* The Publisher refers to the server-side aspect, where the messages are being sent. For the purpose of this tutorial, we will use a simple example that sends the current date and time as Ably events every two seconds. Here is what our code looks like:
+
+
+blang[javascript].
+    ```[javascript]
+    const ably = require('ably').Realtime;
+    const ablyRealtime = new ably('INSERT-YOUR-API-KEY-HERE')
+    const channel = ablyRealtime.channels.get('example-channel');
+    const newChannel = ablyRealtime.channels.get('new-channel');
+    setInterval(()=> {
+        let d = Date(Date.now());  
+        d = d.toString();
+        channel.publish('event', `Hello, current time is ${d}`)
+        newChannel.publish('event', `Hello, current time is ${d}`)     
+    }, 2000); 
+    ```
+
+blang[python].
+    ```[py]
+    from ably import AblyRest
+    from threading import Timer
+    import datetime
+    client = AblyRest('INSERT-YOUR-API-KEY-HERE')
+    channel = client.channels.get('example-channel')
+    newChannel = client.channels.get('new-channel')
+    def sendEvent():
+        currentDT = datetime.datetime.now()
+        channel.publish('event', "Current date is %s" % (currentDT))
+        newChannel.publish('event', "Current date is %s" % (currentDT))
+        Timer(2.0, sendEvent).start()
+    Timer(2.0, sendEvent).start()
+    ```
+
+blang[php].
+    ```[php]
+    $client = new Ably\AblyRest('INSERT-YOUR-API-KEY-HERE');
+    $channel = $client->channel('example-channel');
+    $newChannel = $client->channel('new-channel');
+
+    function setInterval($func = null, $interval = 0){
+    if( ($func == null) || (!function_exists($func)) ){
+        throw new Exception('We need a valid function.');
+    }
+    $seconds = $interval * 1000;
+        while(true){
+            call_user_func($func); // Call the function you've defined.
+            usleep( $seconds );
+        }
+    }
+
+    function sendEvent(){
+    $date = new \DateTime('now');
+    $channel->publish('myEvent', "Current date is $date->format('D M d, Y G:i')" );
+    $newChannel->publish('myEvent', "Current date is $date->format('D M d, Y G:i')" );
+    }
+    setInterval('sendEvent', 2000);
+    ```
+
+h2(#setting-up-the-subscriber-with-http-event-streaming). Setting up the subscriber with HTTP Event Streaming
+
+Let us add the code below to our HTML file:
```

Comment:
Please mention where exactly in the HTML file is this code needed to be added. Some readers may be super beginners, it's always better to mention details :)